module/TaskManagement/test/unit/TaskManagement/Controller/RemindersControllerTest.php

```diff
@@ -9,6 +9,7 @@
 use UnitTest\Bootstrap;
 use ZFX\Acl\Controller\Plugin\IsAllowed;
 use TaskManagement\Entity\Task;
+use Zend\Mvc\Router\RouteMatch;
 
 class RemindersControllerTest extends ControllerTest
```

Comment:
E' più opportuno creare 2 differenti TestCase, uno per la notifica 'assignment-of-shares' ed uno per 'add-estimation'